tests/x509/test_x509.py

```diff
@@ -913,6 +913,34 @@ def test_tbs_certificate_bytes(self, backend):
             cert.signature_hash_algorithm,
         )
 
+    @pytest.mark.supported(
+        only_if=lambda backend: backend.signature_hash_supported(
+            hashes.SHA1()
+        ),
+        skip_message="Does not support SHA-1 signature.",
+    )
+    def test_tbs_precertificate_bytes(self, backend):
+        cert = _load_cert(
+            os.path.join("x509", "custom", "post2000utctime.pem"),
+            x509.load_pem_x509_certificate,
+            backend,
+        )
+
+        # This cert doesn't have an SCT list extension, so it will throw a
+        # `ValueError` when we try to retrieve the property
+        with pytest.raises(ValueError):
+            cert.tbs_precertificate_bytes
+
+        public_key = cert.public_key()
+        assert isinstance(public_key, rsa.RSAPublicKey)
+        assert cert.signature_hash_algorithm is not None
+        public_key.verify(
+            cert.signature,
+            cert.tbs_certificate_bytes,
+            padding.PKCS1v15(),
+            cert.signature_hash_algorithm,
+        )
```

Comment:
What is this testing?

Reply:
Previously, this was testing that we didn't modify certs that weren't precerts. But we're changing the implementation to raise a `ValueError` when the extension is missing, so this will become a `pytest.raises` check instead.

Reply:
Pretty sure this can just be deleted, it's testing `tbs_certificate_bytes`, not the precertificate bytes.